lib/cocina/models/file_set_type.rb

```diff
@@ -0,0 +1,72 @@
+# frozen_string_literal: true
+
+module Cocina
+  module Models
+    # This vocabulary defines the types of file sets
+    class FileSetType
+      def self.three_dimensional
+        'http://cocina.sul.stanford.edu/models/resources/3d.jsonld'
```

Comment:
I can't recall... are we leaving `resources` in these URIs?

Reply:
That's a different PR.